WalletWasabi.Backend/Global.cs

```diff
@@ -103,7 +103,8 @@ private async Task InitializeP2pAsync(Network network, EndPoint endPoint, Cancel
 			// We have to find it, because it's cloned by the node and not perfectly cloned (event handlers cannot be cloned.)
 			P2pNode = new(network, endPoint, new(), $"/WasabiCoordinator:{Constants.BackendMajorVersion}/");
 			await P2pNode.ConnectAsync(cancel).ConfigureAwait(false);
-			HostedServices.Register<BlockNotifier>(new BlockNotifier(TimeSpan.FromSeconds(7), RpcClient, P2pNode), "Block Notifier");
+			using var blockNotifier = new BlockNotifier(TimeSpan.FromSeconds(7), RpcClient, P2pNode);
```

Comment:
Same case here. The `HostedServices` container is the one responsible for managing the registered services lifetime.